scripts/xsslint/tests/test_main.py

```diff
@@ -58,11 +57,13 @@ def test_lint_defaults(self):
         Tests the top-level linting with default options.
         """
         out = StringIO()
-        summary_results = SummaryResults()
+        template_linters = self._build_linters()
+        ruleset = _build_ruleset(template_linters)
+        summary_results = SummaryResults(ruleset)
```

Comment:
Would some or all of this work in setUp()?

Reply:
Probably, I'll consider moving it there.

Reply:
Changed.